content/docs/intro/concepts/stack.md

```diff
@@ -4,14 +4,16 @@ meta_desc: An in depth look at Pulumi Stacks and their usage.
 menu:
   intro:
     parent: concepts
-    weight: 3
+    weight: 2
 
 aliases: ["/docs/reference/stack/"]
 ---
 
-Every Pulumi program is deployed to a **stack**.  A stack is an isolated, independently [configurable]({{< relref "config" >}})
+Every Pulumi program is deployed to a **stack**. A stack is an isolated, independently [configurable]({{< relref "/docs/intro/concepts/config" >}})
 instance of a Pulumi program. Stacks are commonly used to denote different phases of development (such as **development**, **staging** and **production**) or feature branches (such as **feature-x-dev**, **jane-feature-x-dev**).
 
+A project can have as many stacks as you need. By default, Pulumi creates a stack for you when you start a new project. The stack recognizes the current directory as the default name for your program.
```

Comment:
Not quite right -- the current directory is used to name the _project_. The default stack name is `dev`, and is unrelated to the current directory name.

```suggestion
A project can have as many stacks as you need. By default, Pulumi creates a stack for you when you start a new project.
```

I do think it's worth leaving in (or otherwise mentioning) that `pulumi new` offers to create a new stack with the default name of `dev`, though. Was that removed intentionally?

Reply:
I don't see that last part mentioned in the current stack docs, but I'll add it in.